### PR TITLE
Update to Angular 10

### PR DESCRIPTION
**Pull request by aTurmo**
pr discovery module : https://github.com/vidal-community/ng2-discovery/pull/13
pr ng-toolbar : https://github.com/softwarevidal/ng-toolbar/pull/17

### CODE REVIEW

**Comment by nicokosi** [comment]
I just made this PR a [draft](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests) (it's available for public repos) instead of adding a `wip` label. :nerd_face:

File: src/polyfills.ts [hunk 1]
```diff
@@ -41,8 +41,8 @@
 
 
 /** Evergreen browsers require these. **/
-import 'core-js/es6/reflect';
-import 'core-js/es7/reflect';
+// import 'core-js/es6/reflect';
```
Comment (s-petit) [inline src/polyfills.ts]: commentaire ?

**Review by s-petit — APPROVED** [review]
:shrug:

**Comment by nicokosi** [comment]
Faire un(des) joli(s) message(s) de commit. :pray:

**Review by nicokosi — APPROVED** [review]
:+1:

**Pull request merged**